scripts/configure_pyproject_toml.py

```diff
@@ -122,6 +122,18 @@ def parse_args():
         "scipy-openblas32>=0.3.26",
     ]
 
+    if backend == "lightning_tensor":
+        dependencies += ["cutensor-cu12>=2.0.2"]
+
+    if backend in ("lightning_gpu", "lightning_tensor"):
+        dependencies += [
```

Comment:
Shall we add `custatevec-cu12` to the list of dependencies?

Reply:
added